.travis.yml

```diff
@@ -23,9 +24,21 @@ stages:
 jobs:
   include:
   - stage: Run tests
+    env:
+    - AWS_ACCESS_KEY_ID: dummy
+    - AWS_SECRET_ACCESS_KEY: dummy
+    - AWS_DEFAULT_REGION: eu-north-1
     script:
     - pipenv install --dev
-    - pipenv run pytest -v
+    - pipenv run pytest -v -m "not integration"
+    - git clone https://github.com/codelabsab/timereport-api.git ../timereport-api
```

Comment:
Should we always run integration tests for all commits on all branches?

Reply:
Good question, I think we should!
We still have fast tests (full run takes < 3 minutes on travis) and I always prefer fast feedback if I make a mistake. Preferably feedback on local tests (I get this here) but at least feedback from CI before my PR is reviewed. If we decide that this is too slow I think we should start using branch filters, for example not running integration tests for branches starting with `wip/` (or something similar) or not running integration tests for commits with special tag in the message (think `[skip ci]` for integration tests only).

Does that make sense?

Reply:
Sure, let's go with this for now and evaluate 👍